app/views/miq_policy/_form_expression.html.haml

```diff
@@ -1,25 +1,29 @@
 #form_expression_div
   - if @expkey == :expression
-    %h3= _("Expression (Choose an element of the expression to edit)")
+    %h3= _("Choose an element of the expression to edit")
```

Comment:
Just to make sure: you removed the `h3` from the `else` branch but not here. Don't you want to move this line above the `if`? Or was the `h3` in the `else` unnecessary?